index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <!-- header -->
+    
+    <div class="container">  <!-- wrapper -->
+        
+        <header>
+            <nav class="navbar">
+                <a class="logo" href="#"><img src="./images/Logo.png" alt="Image-logo missing"></a>
+                <a class="bar" href="#"><img src="./images/Union.png" alt="Image-bar  missing"></a>
+            </nav>
```

Comment:
- [ ] **Good job on using the ```nav``` tag for semantics reasons, 👍** However, As ```<nav>``` tag defines a set of navigation links. It would be better to remove the ```logo``` in the nav tag to make your codes more professional 😉😉